packages/components/src/components/assets/AssetShareIndicator/AssetShareIndicator.tsx

```diff
@@ -101,15 +195,31 @@ export const AssetShareIndicator = ({
     size = 32,
     percentageShare,
     index,
+    coingeckoId = '',
+    contractAddress,
+    quality = 'medium',
     ...rest
-}: AssetShareIndicatorProps) => (
```

Comment:
I know it wasn't here before but could you add storybook for this file. It's hard to test it without it.

Reply:
Done.